src/NzbDrone.Core/Datastore/TableMapping.cs

```diff
@@ -60,7 +60,7 @@ public static void Map()
                   .Ignore(i => i.Enable)
                   .Ignore(i => i.Protocol)
                   .Ignore(i => i.SupportsRss)
-                  .Ignore(i => i.SupportsSearch)
+                  .Ignore(i => i.SupportsSearch)    
```

Comment:
stray whitespace.

If you're using visual studio, install the EditorConfig extension. It'll read the .editorconfig in the root and configure the IDE for a few settings such as tab,spaces, eol etc

Reply:
Thx for the tip, removed the whitespace